src/itemTracker/GridTracker.tsx

```diff
@@ -20,6 +21,14 @@ export default function GridTracker({ width }: { width: number }) {
         dispatch(clickItem({ item: 'Extra Wallet', take: false }));
     };
 
+    const handleExtraWalletDrag = (event: React.DragEvent<HTMLDivElement>) => {
+        event.dataTransfer.setData('text/plain', 'Extra Wallet');
+        event.dataTransfer.effectAllowed = 'move';
```

Comment:
Maybe the code that sets and gets item data can be moved to a separate file, like `utils/DragAndDrop`, since quite a bunch of components can set and read this data.